deploy/docker-run.sh

```diff
@@ -4,10 +4,5 @@ set -u   # crash on missing env variables
 set -e   # stop on any error
 set -x
 
-echo Collecting static files
-python manage.py collectstatic --no-input
-
-python manage.py createcachetable
```

Comment:
Why are you moving collectstatic to the dockerfile? You are then missing the flags set above (crash on missing envs, stop on any error and debugging). Is this what you want?